CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+1. [General] Workaround of lack of tiller input for nose wheel steering (NWS) - @Villason (Villason)
```

Comment:
Please don't create a new changelog, [use the existing one](https://github.com/Villason/a32nx/blob/nws-pedals/tiller-combo/.github/CHANGELOG.md)

```suggestion
```

Reply:
Done.